src/app/shared/palette/payment/case-payment-history-viewer-field.component.ts

```diff
@@ -0,0 +1,25 @@
+import { AbstractFieldReadComponent } from '../base-field/abstract-field-read.component';
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'ccd-case-payment-history-viewer-field',
+  templateUrl: 'case-payment-history-viewer-field.html',
+  styleUrls: ['case-payment-history-viewer-field.scss']
+})
+export class CasePaymentHistoryViewerFieldComponent extends AbstractFieldReadComponent {
+
+  public static readonly PAYMENTS_BASE_URL_KEY = 'payments_url';
+  public static readonly CASE_REFERENCE_KEY = 'case_reference';
```

Comment:
case reference is very generic concept. This key will probably be used by other components soon.
Shall we move it outside of CasePaymentHistoryViewerFieldComponent?